src/media/mod.rs

```diff
@@ -208,25 +217,24 @@ impl ::std::fmt::Display for MediaCategory {
     }
 }
 
-/// A builder struct that allows you to build up parameters to a media upload before initiating it.
+/// A builder that provides mechanism to create upload and customize it.
 ///
-/// `UploadBuilder` is your entry point to uploading media to Twitter. This allows you to configure
-/// an upload and set the proper metadata, so that when you `call` it to begin the upload proper,
-/// the resulting [`UploadFuture`] can take care of all the underlying details of the upload itself.
+/// `UploadBuilder` is entry point to uploading media to Twitter.
+///  It allows you to configure an upload and set the proper metadata.
 ///
 /// [`UploadFuture`]: struct.UploadFuture.html
 ///
 /// To begin setting up an upload, call `new` with your data and its media type. (Convenience
 /// functions to create `Mime` instances for types Twitter is known to accept are available in the
-/// [`media_types`] module.) After that, you can configure the upload, and finally start the
-/// process with `call`. A basic example of using `UploadBuilder` to upload an image and attach it
+/// [`media_types`] module.) With that, you can configure the upload, and finally start the
+/// process using `call` method. See a basic example of using `UploadBuilder` to upload an image and attach it
```

Comment:
* "using the `call` method."
* The flow of the last sentence here before your change was "A basic example... is in the module documentation" which your change renders ungrammatical. If you change the end of the sentence to take out the "is" then it would work.

Reply:
- Ok
- Opps didn't notice this 'is'. Removed.